### PR TITLE
Enables the indexing the barcode property for EphemeraBoxes and EphemeraFolders

### LINKED ISSUE #473: Ensure search by barcode works

### PR DESCRIPTION
**Pull request by jrgriffiniii**
Resolves #473

### CODE REVIEW

File: app/indexers/ephemera_box_indexer.rb [hunk 1]
```diff
@@ -11,7 +11,8 @@ def to_solr
     {
       "title_tesim" => [decorated.title],
       "title_ssim" => [decorated.title],
-      "title_tsim" => [decorated.title]
+      "title_tsim" => [decorated.title],
+      "barcode_ssim" => [decorated.barcode]
```
Comment (tpendragon) [inline app/indexers/ephemera_box_indexer.rb]: I think this is indexed already.

Comment (jrgriffiniii) [reply app/indexers/ephemera_box_indexer.rb]: It was originally failing without it, I will add another commit.

**Comment by jrgriffiniii** [comment]
@tpendragon : It does appear to fails if it isn't explicitly indexed: https://circleci.com/gh/pulibrary/figgy/1005

**Comment by jrgriffiniii** [comment]
The indexing has been fixed and this is prepared for another review please.

**Pull request merged**